.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: OPENWISP WIFI LOGIN PAGES CI BUILD
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: -master
+
+jobs:
+  build:
+    name: Python==${{ matrix.python-version }}
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
```

Comment:
We need python only for qa-checks. Right?
Therefore there is no point checking for different python versions